python/tests/conftest.py

```diff
@@ -17,12 +17,15 @@
 
 import os
 import zipfile
-from pathlib import Path
+from pathlib import Path, PosixPath
 
 import pytest
+from _pytest.fixtures import SubRequest
 
+from tests.table.table_path import get_table_path
 
-def _extract_testing_table(zip_file_path, target_path) -> str:
+
+def _extract_testing_table(zip_file_path: Path, target_path: PosixPath) -> str:
```

Comment:
PosixPath restricted this from running on windows? we'll need to support test and build on windows too

Reply:
Oh. Got it. Lemme figure out how to test this